Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -756,9 +818,9 @@ HRESULT _DWriteCreateFontFaceWithName(CFStringRef name, IDWriteFontFace** outFon
     DWRITE_FONT_STRETCH stretch = DWRITE_FONT_STRETCH_NORMAL;
     DWRITE_FONT_STYLE style = DWRITE_FONT_STYLE_NORMAL;
 
-    __InitDWriteFontPropertiesFromName(name, &weight, &stretch, &style);
+    CFStringRef familyName;
+    __InitDWriteFontPropertiesFromName(name, &weight, &stretch, &style, &familyName);
```

Comment:
It's owned by the dictionary which has all of the family names

---
In reply to: [79964582](https://github.com/Microsoft/WinObjC/pull/1021#discussion_r79964582) [](ancestors = 79964582)